src/Tizen.NUI/src/public/Color.cs

```diff
@@ -184,8 +184,18 @@ internal Color(ColorChangedCallback cb, Color other) : this(cb, other.R, other.G
         /// <summary>
         /// The red component.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Color(...) constructor.
```

Comment:
```suggestion
        /// The setter is deprecated in API8 and will be removed in API10. Use the new Color(...) constructor.
```

Reply:
thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118